### DISCUSSION TITLE
Does Rust9x support mingw?

### DISCUSSION THREAD

**Discussion by K4sum1** [rust9x/rust#23] [Q&A]
A lot of modern programs also support compiling with mingw. From what I can tell the latest mingw can compile programs compatible with XP. Just using mingw would be a lot easier than trying to get it working on older msvc versions. I went through the wiki and can't find any mentions of mingw.

**Comment by seritools** [comment]
It does not support mingw targets at the moment.

I'm open to contributions there, but won't be implementing it myself. Added #24 to track it

**Comment by seritools** [comment]
It does now! https://github.com/rust9x/rust/releases/tag/rust9x-1.98-beta-v1
